dxr/plugins/python/indexers.py

```diff
@@ -153,30 +227,35 @@ def __init__(self, path, contents, plugin_name, tree, python_path,
         self.derived_classes = derived_classes
         self.names = names
 
+        self.module_name = path_to_module(self.python_path, self.path)
+
     def is_interesting(self):
         return is_interesting(self.path)
 
     def needles_by_line(self):
+        self.analyze_tokens()
+
+        visitor = IndexingNodeVisitor(self)
+        syntax_tree = ast.parse(self.contents.encode('utf-8'))
```

Comment:
I wish I wrote a comment or test about this, but I seem to remember running into some issue where self.contents caused ast.parse to fail. But I cannot for the life of me remember what it was.

Maybe I'll remove it and see if I can come up with anything? `test_unicode.py` maybe?